frontend/src/components/Answer/Answer.tsx

```diff
@@ -12,6 +12,8 @@ import { parseAnswer } from "./AnswerParser";
 import ReactMarkdown from "react-markdown";
 import remarkGfm from "remark-gfm";
 import supersub from 'remark-supersub'
+import {Prism as SyntaxHighlighter} from 'react-syntax-highlighter';
+import { nord } from 'react-syntax-highlighter/dist/esm/styles/prism';
```

Comment:
you missed to include "react-syntax-highlighter" & "@types/react-syntax-highlighter" in package.json